aehmc/proposals.py

```diff
@@ -87,15 +87,12 @@ def progressive_uniform_sampling(
     state, energy, weight, _ = proposal
     new_state, new_energy, new_weight, _ = new_proposal
 
-<<<<<<< HEAD
```

Comment:
🧐

Reply:
Yeah I wish the code got to that point in the warmup.